### PR TITLE
Hardcoded clickable views

### PR DESCRIPTION
**Pull request by camilleldn**

### CODE REVIEW

File: app/views/start/index.html.erb [hunk 1]
```diff
@@ -8,3 +8,26 @@
   <li>skills and qualifications required</li>
   <li>tasks and hours</li>
 </ul>
+ 
+<%= form_tag("/select_soc_codes", method: "get") do %>
```
Comment (gjvis) [inline app/views/start/index.html.erb]: Does this create a CSRF token and UTF8 field on the form? I don't think we want that for a form with method `get...`

File: app/views/start/index.html.erb [hunk 2]
```diff
@@ -8,3 +8,26 @@
...
+  <div class="form-group ">
+    <label for="id_sector_form-sector_0">
+      <span class="form-label">Work, skills and interests</span>
+    </label>
+    <ul class="multi-entry-text-input">
+    <li>
+      <%= text_field_tag(:keyword_1, nil, class: 'form-control') %>
```
Comment (gjvis) [inline app/views/start/index.html.erb]: Plain HTML is easier to understand, I don't thin we should use rails helpers unless they're adding value (like handling validation, etc.)

File: app/views/start/select_soc_codes.html.erb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    </section>
+  <% end %>
+
+  <div class="form-group">
+    <button type=submit class=button>Next</button>
+  </div>
+<% end %>
+
+<footer>
+  <a href="/" class="link-back">start again</a>
```
Comment (gjvis) [inline app/views/start/select_soc_codes.html.erb]: `href="<%= root_url %>"`?

**Comment by gjvis** [comment]
Some very minor comments and questions, otherwise :+1:

**Comment by camilleldn** [comment]
Minor comments addressed, merging this so I can rebase the rest of my work that makes this actually functional :-D

**Comment by gjvis** [comment]
whoop whoop

**Pull request merged**